README.md

```diff
@@ -1 +1,63 @@
-# Global-Summit
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TPPS Conference (Microverse captsone 1)
+
+> This website (purely fictional) is 
```

Comment:
- [ ] Good job so far 👏  Your website looks amazing 👏 🔥  However a good project should have a good professional README. That's why I suggest that you add a brief description of your project here. For example, you can say that It is a landing page for a conference called TPPS Conference.. and so on 😉